doc/nrf/libraries/caf/buttons_pm_keep_alive.rst

```diff
@@ -0,0 +1,26 @@
+.. _caf_buttons_pm_keep_alive:
+
+CAF: Buttons power manager keep alive module
```

Comment:
```suggestion
CAF: Power manager keep alive module for buttons
```
I guess this module is more related to power manager than to buttons? *Theoretically* speaking, there could be a similar module for LEDs?